### DISCUSSION TITLE
Multi-Language Model - which languages are included ?

### DISCUSSION THREAD

**Discussion by tamsial** [explosion/spaCy#11950] [Help: Model Advice] [answered]
Hello, everyone,

for the multi-language models it is not specified which languages ​​this model contains. Everywhere it says that all languages ​​are included, which are they exactly? Specifically the xx_ent_wiki_sm model.

Thanks in advance!

**Comment by polm** [comment, marked as answer]
If you look at the [model page](https://spacy.io/models/xx) for this you can see it's based on WikiNER, and looking at the paper linked from there you can see these are the languages covered:

> English, German, French, Polish, Italian, Spanish, Dutch, Portuguese and Russian
